src/main/java/frc/robot/subsystems/drivetrain/Drivetrain.java

```diff
@@ -1,10 +1,153 @@
-package frc.robot.subsystems.drivetrain;
-
-import com.ctre.phoenix.motorcontrol.can.TalonFX;
-
-public class Drivetrain {
-    private final TalonFX drivetrainMaster1=new TalonFX(1);
-    private final TalonFX drivetrainMaster2=new TalonFX(2);
-    private final TalonFX drivetrainMaster3=new TalonFX(3);
-    private final TalonFX drivetrainMaster4=new TalonFX(4);
-}
+package frc.robot.subsystems.drivetrain;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.Robot;
+import frc.robot.subsystems.UnitModel;
+
+/**
+ * Build 4 motors, 2 for each side.
+ */
+public class Drivetrain extends SubsystemBase {
+    private final TalonFX masterRight = new TalonFX(Ports.Drivetrain.MASTER_RIGHT);
+    private final TalonFX slaveRight = new TalonFX(Ports.Drivetrain.SLAVE_RIGHT);
+    private final TalonFX masterLeft = new TalonFX(Ports.Drivetrain.MASTER_LEFT);
+    private final TalonFX slaveLeft = new TalonFX(Ports.Drivetrain.SLAVE_LEFT);

```

Comment:
```suggestion
    private final TalonFX rightMaster = new TalonFX(Ports.Drivetrain.MASTER_RIGHT);
    private final TalonFX rightSlave = new TalonFX(Ports.Drivetrain.SLAVE_RIGHT);
    private final TalonFX leftMaster = new TalonFX(Ports.Drivetrain.MASTER_LEFT);
    private final TalonFX leftSlave= new TalonFX(Ports.Drivetrain.SLAVE_LEFT);
```
And the left motors should come before the right ones.